tequilapi/endpoints/node.go

```diff
@@ -78,6 +80,32 @@ func (ne *NodeEndpoint) MonitoringAgentStatuses(c *gin.Context) {
 	utils.WriteAsJSON(contract.MonitoringAgentResponse{Statuses: res}, c.Writer)
 }
 
+// GetProviderSessions A list of sessions metrics during a period of time
+// swagger:operation GET /node/provider/sessions GetProviderSessions
+// ---
+// summary: Provides Node sessions data during a period of time
+// description: Node sessions metrics during a period of time
+// parameters:
+//   - in: query
+//     name: range
+//     description: period of time ("1d", "7d", "30d")
```

Comment:
Is it possible to have range as `[dateA, dateB]` ?

Reply:
[Screencast from 08-11-2022 08:12:48 AM.webm](https://user-images.githubusercontent.com/20511486/184068432-72322a39-152f-4899-94a2-88ebda7bd1cd.webm)

Reply:
It's possible, but we don't want to give too much freedom to users since it might give us too big load.

Reply:
what if you would limit this range to say max 7 day period? or something like that

Reply:
@mdomasevicius as we talked, free 7-day period might be abused as well. 
Let's stay with static limits for the last period of 1, 7, and 30 days now, and maybe later we will find a better way.

Reply:
discussed with Dmitry and our PO - let's keep this 1, 7, 30 day period for now